examples/hitl/rearrange_v2/config/lang_rearrange_llmspot_guihumanoid.yaml

```diff
@@ -0,0 +1,90 @@
+# @package _global_
+
+defaults:
+  - /training@ : base_train
+  - habitat_conf/task : rearrange_easy_multi_agent
+  - world_model@world_model: gt_graph
+  - /trajectory@trajectory : trajectory_logger
+  # single agent planner setup
+  - /evaluation: decentralized_evaluation_runner_single_agent
+  - /wandb_conf@                : own
+  - language_rearrange_multi_agent_llm_gui
+  - hitl_defaults
+  - override /instruct@evaluation.agents.agent_0.planner.plan_config.instruct: few_shot_decentralized_robot_partner_aware_v1
+  - override /llm@evaluation.agents.agent_0.planner.plan_config.llm: openai_chat
+  - _self_
+
+evaluation:
+  agents:
+    agent_0:
+      planner:
+        plan_config:
+          replanning_threshold: 30
+habitat_llm:
+  enable: True
+    # agent_1:
+    #   gui_controlled: True
+world_model:
+  partial_obs: True
+habitat:
+  # various config args to ensure the episode never ends
+  environment:
+    max_episode_steps: 0
+    iterator_options:
+      # For the demo, we want to showcase the episodes in the specified order
+      shuffle: False
```

Comment:
We'll want to pull [those changes](https://github.com/facebookresearch/habitat-lab/pull/1935) here.